compose/local/django/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-slim-buster
+FROM python:3.8-slim
```

Comment:
Should we go ahead and use 3.9 while we're at it like we do for the other apps?

Reply:
We have a few apps that are still on python 3.8. That's a new enough version that I don't feel _too_ bad about leaving it...but we could definitely stand to upgrade down the line!